CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # Apache Thrift Changelog
 
+## 0.15.0
+
+### Deprecated Languages
+
+### Removed Languages
+
+### Breaking Changes
+
+### Go
+
+- [THRIFT-5369](https://issues.apache.org/jira/browse/THRIFT-5369) - No longer pre-allocation the whole container (map/set/list) in compiled go code to avoid huge allocations on malformed messages.
```

Comment:
nit
```suggestion
- [THRIFT-5369](https://issues.apache.org/jira/browse/THRIFT-5369) - No longer pre-allocating the whole container (map/set/list) in compiled go code to avoid huge allocations on malformed messages.
```